### PR TITLE
feature #2: allow empty or nil accept header in test spec

### PR DESCRIPTION
**Pull request by missedone**

### CODE REVIEW

File: src/github.com/eriklupander/gotling/httpaction.go [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
-	Variable string `yaml:"variable"`
-	Index string `yaml:"index"`
+    Jsonpath string `yaml:"jsonpath"`
+    Variable string `yaml:"variable"`
+    Index    string `yaml:"index"`
 }
 
 func NewHttpAction(a map[interface{}]interface{}) HttpAction {
     var valid bool = true
-    if a["accept"] == nil || a["accept"] != "json" {
```
Comment (missedone) [inline src/github.com/eriklupander/gotling/httpaction.go]: remove the validation for accept header

File: src/github.com/eriklupander/gotling/httpaction.go [hunk 2]
```diff
@@ -67,14 +64,19 @@ func NewHttpAction(a map[interface{}]interface{}) HttpAction {
         responseHandler.Variable = response["variable"].(string)
         responseHandler.Index = response["index"].(string)
     }
+
+    accept := "text/html,application/json,application/xhtml+xml,application/xml,text/plain"
+    if a["accept"] != nil && len(a["accept"].(string)) > 0 {
```
Comment (missedone) [inline src/github.com/eriklupander/gotling/httpaction.go]: override the default accept header only if it is specified in the test spec

**Comment by eriklupander** [comment]
Thanks!

File: src/github.com/eriklupander/gotling/httpreq.go [hunk 3]
```diff
@@ -1,26 +1,25 @@
 package main
```
Comment (missedone) [inline src/github.com/eriklupander/gotling/httpreq.go]: no function change in this file, 
just want to confirm with you about the format style, for example:
> Indentation
We use tabs for indentation and gofmt emits them by default. Use spaces only if you must.

you can see the the original code mixed up tabs and spaces, for now, i use spaces over tabs which make few changes.

actually i'd prefer with the gofmt way...

**Pull request merged**